client/profile-wizard/steps/business-details.js

```diff
@@ -100,9 +113,21 @@ class BusinessDetails extends Component {
 			revenue,
 			used_platform: otherPlatform,
 			used_platform_name: otherPlatformName,
-			install_facebook: values[ 'facebook-for-woocommerce' ],
-			install_mailchimp: values[ 'mailchimp-for-woocommerce' ],
-			install_google_ads: values[ 'kliken-marketing-for-google' ],
+			install_woocommerce_services: businessExtensions.includes(
+				'facebook-for-woocommerce'
```

Comment:
Should be `'woocommerce-services'`.

Reply:
Thanks for pointing this out! https://github.com/woocommerce/woocommerce-admin/pull/5614